path_dynamic_creds.go

```diff
@@ -15,6 +16,30 @@ import (
 	"golang.org/x/text/encoding/unicode"
 )
 
+func (b *backend) pathDynamicCredsCreate() []*framework.Path {
+	return []*framework.Path{
+		{
+			Pattern: path.Join(dynamicCredPath, framework.MatchAllRegex("name")),
```

Comment:
Is `MatchAllRegex` definitely correct here? I'm not sure if more is happening under the hood, but the regex looks like it allows slashes too. I know this is pre-existing though, so no need to actually address in this PR.

Reply:
Good find, it wasn't correct. I've updated this to use `framework.GenericNameRegex` in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/40/commits/b1426f2ee4809afff541a7d27ed2fd872ade1894.